src/lib/components/pager/terra-pager.component.scss

```diff
@@ -10,21 +10,20 @@
     --input-wrapper-margin-bottom: 0;
     --input-width: 43px;
     --btn-sm-padding: 0 7px;
-    --toolbar-border-radius: var(--border-radius);
 
     .pager-inner-wrapper
     {
         display: block;
         flex-wrap: initial;
         justify-content: initial;
-        background-color: var(--toolbar-background-color, var(--color-structure-0));
-        padding: var(--toolbar-padding, var(--space-sm));
-        margin: var(--toolbar-margin, var(--space-container));
-        border-radius: var(--toolbar-border-radius, 0 0 var(--border-radius) var(--border-radius));
-        border-bottom: var(--toolbar-border-bottom, (none));
+        background-color: var(--pager-background-color, var(--color-structure-0));
+        padding: var(--pager-padding, var(--space-sm));
+        margin: var(--pager-margin, 0 var(--space-xs) var(--space-sm) var(--space-xs));
```

Comment:
Weshalb hat sich hier der Wert verändert?

Reply:
weil das in der tabelle vorher definiert war und mit Saman abgesprochen wurde, dass das der Standartwert ist, da der Pager nirgends verwendet wird